hack/configure_environment.sh

```diff
@@ -1,4 +1,7 @@
 #!/bin/bash
```

Comment:
@jordigilh shouldn't this script be named `setup.sh`? at least this is what the documentation refers to.